docs/ace_adm.rst

```diff
@@ -0,0 +1,55 @@
+ACE & ADMs
```

Comment:
Ampersand in the title might cause rendering issues, suggest to just use "and" here.